src/garage/tf/samplers/__init__.py

```diff
@@ -3,7 +3,10 @@
     OffPolicyVectorizedSampler)
 from garage.tf.samplers.on_policy_vectorized_sampler import (
     OnPolicyVectorizedSampler)
+from garage.tf.samplers.ray_batched_sampler import (RaySamplerTF,
+                                                    SamplerWorkerTF)
 
 __all__ = [
-    'BatchSampler', 'OffPolicyVectorizedSampler', 'OnPolicyVectorizedSampler'
+    'BatchSampler', 'OffPolicyVectorizedSampler', 'OnPolicyVectorizedSampler',
+    'RaySamplerTF', 'SamplerWorkerTF'
```

Comment:
there's no reason to name these `FooTF`, because they are already in the tree `garage.tf`. 

I didn't see a anything TF-specific in these classes. Why not put them in `garage.sampler`?